tools/ci.sh

```diff
@@ -234,14 +234,14 @@ CI_UNIX_OPTS_SYS_SETTRACE=(
     MICROPY_PY_BTREE=0
     MICROPY_PY_FFI=0
     MICROPY_PY_USSL=0
-    CFLAGS_EXTRA="-DMICROPY_PY_SYS_SETTRACE=1"
+    CFLAGS_EXTRA="-DMICROPY_PY_SYS_SETTRACE=1 -DMICROPY_PERSISTENT_CODE_SAVE_FILE=1"
```

Comment:
Shouldn't be needed, should be catched by unix/i386/x86_64 ?

Reply:
should but the CI failed for some reason.

Reply:
Sorry muissed that, probably the definitions need to go in py/mpconfig.h